angular-ngrx-scss/package.json

```diff
@@ -18,55 +18,55 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~13.1.0",
-    "@angular/common": "~13.1.0",
-    "@angular/compiler": "~13.1.0",
-    "@angular/core": "~13.1.0",
-    "@angular/forms": "~13.1.0",
-    "@angular/platform-browser": "~13.1.0",
-    "@angular/platform-browser-dynamic": "~13.1.0",
-    "@angular/router": "~13.1.0",
-    "@ngrx/effects": "^13.0.2",
-    "@ngrx/store": "^13.0.2",
-    "@ngrx/store-devtools": "^13.0.2",
+    "@angular/animations": "16.1.7",
+    "@angular/common": "16.1.7",
+    "@angular/compiler": "16.1.7",
+    "@angular/core": "16.1.7",
+    "@angular/forms": "16.1.7",
+    "@angular/platform-browser": "16.1.7",
+    "@angular/platform-browser-dynamic": "16.1.7",
+    "@angular/router": "16.1.7",
+    "@ngrx/effects": "16.1.0",
+    "@ngrx/store": "16.1.0",
+    "@ngrx/store-devtools": "16.1.0",
     "@primer/octicons": "^16.3.1",
```

Comment:
Can we go ahead and lock all the package versions in this file? Looks like most of the Angular ones are locked now, but we should be consistent and just lock everything to a specific version here.